content/_index.en.md

```diff
@@ -7,7 +7,8 @@ hero:
   subtitle: Open Terms Archive publicly records every version of the terms of digital services to enable democratic oversight.
   link: "[Discover how that makes a difference →]({{< relref path=\"/impact\" >}})"
 collections:
-  title: We establish partnerships to track the terms of many industries, in several languages and jurisdictions
+  title: Featured collections
```

Comment:
When seeing the result, I don't find it relevant to have a title at all.

Reply:
I tested it without title and with the text you suggest and I approve

Reply:
Ok